tools/c7n_azure/c7n_azure/function_package.py

```diff
@@ -47,6 +52,7 @@ def create_zinfo(self, file):
 
 class FunctionPackage(object):
     log = logging.getLogger('custodian.azure.function_package.FunctionPackage')
+    disable_wait_for_build = False
```

Comment:
can we skip the double negatives (disable false), ie. just simplify naming to just wait_for_build